storage/s3/src/test/java/io/aiven/kafka/tieredstorage/storage/s3/S3StorageConfigTest.java

```diff
@@ -108,7 +109,8 @@ void configWithStaticCredentials() {
             "s3.region", region.id(),
             "s3.endpoint.url", MINIO_URL,
             "aws.access.key.id", username,
-            "aws.secret.access.key", password);
+            "aws.secret.access.key", password,
+            S3StorageConfig.DISABLE_AWS_CERT_CHECKING_CONFIG, "true");
```

Comment:
nit: could we use a string instead of a field as other configs? We do this to enforce config names do not change inadvertently.

Reply:
sure